app/controllers/users/registrations_controller.rb

```diff
@@ -1,5 +1,11 @@
 module Users
   class RegistrationsController < Devise::RegistrationsController
+
+    def edit
+      authorize User.find(params[:id]), :edit_password?
```

Comment:
Cool, this looks good. We should do the same in the `update` action. Arguably we should have `update_password?` policy method for that one, but we can get away with reusing this one.